### PR TITLE
docs(ce-2.0/go) Go language support

### PR DESCRIPTION
**Pull request by hishamhm**

### CODE REVIEW

**Comment by hishamhm** [comment]
@Kong/team-docs the Netlify preview didn't show up. Is this expected?

**Comment by levfishbluefish** [comment]
> @Kong/team-docs the Netlify preview didn't show up. Is this expected?

I think it is not showing up because the travis tests failed

File: app/2.0.x/go.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+To use Go plugins:
+
+* You need the `go-pluginserver` executable.  The Kong distribution packages
+install it in `/usr/local/bin/`.  If you want it anywhere else, set the
+`go_pluginserver_exe` variable in the Kong configuration file to the full path.
+* Set the `go_plugins_dir` variable in the Kong configuration file to a valid directory.
+The default `"off"` value disables Go support.
+* Add compiled Go plugins to the directory specified in the previous step.
+* Add the plugin settings the usual way, either via Admin API, database or declarative file.
+Refer to the the plugin via it's filename (without the `.so` suffix).
```
Comment (gszr) [inline app/2.0.x/go.md]:
```suggestion
Refer to the the plugin via its filename (without the `.so` suffix).
```

File: app/2.0.x/go.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+* A Go development environment.
+* The Go-PDK package: `go get github.com/Kong/go-pdk`
+
+## Development
+
+To write a Kong plugin in Go, you need to:
+
+* Define a structure type to hold configuration.
+* Write a `New()` function to create instances of your structure.
+* Add methods on that structure to handle events
```
Comment (gszr) [inline app/2.0.x/go.md]:
```suggestion
* Add methods on that structure to handle events.
```

File: app/2.0.x/go.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+mismatch, you can recompile the `go-pluginserver` executable:
+
+```bash
+# git clone https://github.com/Kong/go-pluginserver
+# cd go-pluginserver
+# make
+```
+
+Point the `go_pluginserver_exe` configuration variable to your new version.
+
```
Comment (gszr) [inline app/2.0.x/go.md]: Could we link to https://github.com/Kong/go-plugins somewhere? It might be useful as a reference.

**Pull request merged**